CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 #
 # Please read The Ultimate Guide to CMake:
 # https://rix0r.nl/blog/2015/08/13/cmake-guide/
-cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
```

Comment:
I think the only things we'll lose here that we may have wanted are:

- `target_link_directories()`
- `target_link_options()`

But we'll be fine without them. We cannot go below 3.12, but fortunately, we don't need to for Windows.

If VS updates to include 3.13, let's bump this back up at that point.

Reply:
Sounds good.